libcaf_core/caf/ipv4_subnet.test.cpp

```diff
@@ -16,6 +21,28 @@ ipv4_subnet operator/(ipv4_address addr, uint8_t prefix) {
   return {addr, prefix};
 }
 
+struct fixture {
```

Comment:
Why aren't we just using the deterministic fixture here with [`serialization_roundtrip`](https://github.com/actor-framework/actor-framework/blob/master/libcaf_test/caf/test/fixture/deterministic.hpp#L634)?